<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Поворот вокруг оси</string>
    </property>
    <widget class="QDoubleSpinBox" name="rotateX">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>x:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rotateButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Поворот</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>y:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="rotateY">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>62</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>z:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="rotateZ">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>62</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Перенос вдоль оси</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>dx:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dx">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>62</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dy">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>62</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dz">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>62</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>dy:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>dz:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moveButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Перенос</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Масштабирование</string>
    </property>
    <widget class="QDoubleSpinBox" name="kx">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>62</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="ky">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>62</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="kz">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>62</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>kx:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ky:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>kz:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scaleButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>141</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Масштабирование</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>591</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить фигуру</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Координаты центра поворота/масштабирования</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>x:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>z:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="xc">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>62</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="yc">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>62</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="zc">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>62</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
